content/docs/command-reference/commit.md

```diff
@@ -25,8 +25,8 @@ next.
 💡 For convenience, a pre-commit Git hook is available to remind you to
 `dvc commit` when needed. See `dvc install` for more info.
 
-Mainly, `dvc commit` provides a way to complete DVC commands that track data
-(`dvc add`, `dvc repro`, `dvc import`, etc.), when they have been used with the
+`dvc commit` provides a way to complete DVC commands that track data (`dvc add`,
```

Comment:
```
Sometimes after executing a stage, we realize that not all of its dependencies
  or outputs are defined in `dvc.yaml`. It is possible to
  [add the missing deps/outs](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage),
  and `dvc commit` may be needed to finalize the remedy (see link).
```

contradicts this. (and probably the previous version as well)

Reply:
I don't see the contradiction, but maybe the way this is written gives that impression? Here's what each use case means:
1. finish add/run/repro --no-commit/exec
2. register and cache missing outputs that you forgot to add to dvc.yaml before (but now fixed)

Reply:
Clarified a little in ffff0a43 but it's not a big change.

Reply:
Contradiction here is that you we state "Mainly" and then clarify "Specifically", but point is completely different from the one in the "Mainly" section. To my mind either introductory section should be completely removed, or generalized if we keep it this way. wdyt?

Reply:
Just to clarify, the the "Mainly" word was deleted in the lines above 🙂

> either introductory section should be completely removed, or generalized

The intro is "Stores the current contents..." which is already a general explanation. That p ends in "We explore the scenarios next" and the desc goes on to the main scenario ("provides a way to complete add/repro --no-commit etc."). Then we have a list of other scenarios but we wanted to use the word "Specifically" there.

So I'm a bit confused, why delete the main scenario? Do you mean move it down to the list? How about reinstating "Other" instead of "Specific"? That was my originally intended structure for this Description (again: intro, main scenario, list of other scenarios).